### PR TITLE
Update documentation page (vi)

### PR DESCRIPTION
**Pull request by hienvd**

### CODE REVIEW

**Comment by stomar** [comment]
1. Fix markup (see notes below)
2. Remove unneccessary comment about cherry-pick from commit message. It's completely irrelevant for project's history, and the referenced commit is not even in master.

You can `amend` your commit and force push your branch.

File: vi/documentation/index.md [hunk 1]
```diff
@@ -21,8 +35,8 @@ tại đây.
   dụng của Ruby.
 
 [RubyMonk][3]
-: Khám phá các thành ngữ của Ruby với phương pháp tương tác trực tuyến. Bạn sẽ
-  học thông qua các bài học và giải quyết những bài tập.
+: Khám phá các thành ngữ của Ruby, học các bài học và giải quyết những bài tập,
+tất cả trên trình duyệt của bạn!
```
Comment (stomar) [inline vi/documentation/index.md]: indent by 2 spaces; same below

these are definition lists

```
Keyword
: text
```

File: vi/documentation/index.md [hunk 2]
```diff
@@ -91,6 +111,11 @@ tại đây.
 [APIdock][18]
 : Tài liệu Ruby, Rails và RSpec với các ghi chú của người sử dụng.
 
+[Omniref: Chú thích Ruby, Rails và tài liệu về Gem][40] : Tài liệu
```
Comment (stomar) [inline vi/documentation/index.md]: markdown wrong; should be a definition list, see above

Comment (hienvd) [inline vi/documentation/index.md]: Thank you. I will fix them soon.

**Comment by stomar** [comment]
I'll reopen after PR is fixed.

**Comment by hienvd** [comment]
@stomar: I fixed the PR

**Comment by stomar** [comment]
I still see the old commit; did you force-push your branch?

**Comment by hienvd** [comment]
I force pushed. You can see here: https://github.com/hienvd/www.ruby-lang.org/commit/cee3daca3d84977e6ecfecb625502f19f9768443. I don't know why the PR is not updated, maybe it is being closed?

**Comment by stomar** [comment]
ok, I learned just now that re-opening PR doesn't work with force-pushed branches... a new PR is necessary; sorry.

**Comment by hienvd** [comment]
So, I will open a new PR. Thank you for your support :)

**Pull request closed without merging**